### DISCUSSION TITLE
Server Watchdog crashed

### DISCUSSION THREAD

**Discussion by DonorKrovi** [FabricMC/fabric-api#3418] [Mod Dev Support] [answered]
### Description

I noticed that my server often crashes due to a Watchdog error. Can you give me some guidance on how to identify the problem? This situation has left me puzzled.

Crash Reports:
https://mclo.gs/oyUypy4
https://mclo.gs/9C8zoqE
https://mclo.gs/8csjgyY

**Comment by Linguardium** [comment]
you have 363000 entities it is processing (the area effect clouds...). that will bring any server to its knees

**Reply by DonorKrovi** [reply]
You don't know the method of limiting this?

**Reply by Linguardium** [reply, marked as answer]
find what makes them (maybe a specific cobblemon? maybe something else?) and then disable whatever that is. if the mod doesnt have a config, remove whatever mod it is
